crt_portal/cts_forms/forms.py

```diff
@@ -1290,7 +1290,10 @@ class BulkActionsForm(Form, ActivityStreamUpdater):
     district = ChoiceField(
         label='Judicial district',
         widget=ComplaintSelect(
-            attrs={'class': 'text-uppercase crt-dropdown__data'},
+            attrs={
+                'class': 'text-uppercase crt-dropdown__data',
+                'disabled': 'disabled'
```

Comment:
With this disabled, a form submission clears any existing value in `Report.district`.

Reply:
Yes! That's right, disabled == read-only and thus browsers never submit this value. D'oh. Fixed.